sample/src/main/java/com/yalantis/ucrop/sample/SampleActivity.java

```diff
@@ -169,7 +172,14 @@ private void startCropActivity(@NonNull Uri uri) {
 
         uCrop = basisConfig(uCrop);
         uCrop = advancedConfig(uCrop);
+        uCrop.start(SampleActivity.this);
+    }
+
+    private void startCropActivity(@NonNull String url) {
```

Comment:
no need for this, library should accept only Uri (if user has url string - Uri.parse(url))